backend/app/src/anonymous-user/anonymous-use.service.ts.sample

```diff
@@ -0,0 +1,57 @@
+// import { PrismaClient } from "@prisma/client";
+
+// const prisma = new PrismaClient();
+
+// async function main()
+// {
+// 	// lecture de la base de donnee a partir de prisma
+// 	const allUsers = await prisma.user.findMany();
+
+// 	// creation des Array en memoire, ici juste un console.log
+// 	console.log(allUsers)
+// }
+
+// main()
+// 	.catch(e =>
+// 	{
+// 		throw e;
+// 	})
+// 	.finally(async () =>
+// 	{
+// 		await prisma.$disconnect();
+// 	});
+
```

Comment:
This part up there is commented, should we keep it ?

Reply:
This sample file will be removed yep